R/data-module.R

```diff
@@ -209,19 +209,15 @@ setMethod("labels", "DataModule", function(object) {
     .labels(modulePath(object))
 })
 
+
 #' @rdname DataModule
-#'
-#' @export
-setMethod("groups", "DataModule", function(object) {
-    validObject(object)
-    .groups(modulePath(object))
-})
+groups <- function(object, ...) UseMethod("groups")
```

Comment:
I would use here also `x` because otherwise we get a problem/error (or warning) for the `dplyr::groups` function, no?